### DISCUSSION TITLE
E0 for Iodine vs Bromine

### DISCUSSION THREAD

**Discussion by JSLJ23** [ACEsuit/mace#474] [Q&A] [answered]
I was looking into the training scripts for MACE-OFF23 and I realised that for the E0 string that was specified as an argument, the E0 value for Iodine, which I assumed had they key of 53 was much lower than that of Bromine, which seems to have the key of 35.
```
--E0s="{35: -70045.28385080204, 6: -1030.5671648271828, 17: -12522.649269035726, 9: -2715.318528602957, 1: -13.571964772646918, 53: -8102.524593409054, 7: -1486.3750255780376, 8: -2043.933693071156, 15: -9287.407133426237, 16: -10834.4844708122}"
```
Is there a reason why the E0s are increasing in line with the atomic numbers from 1 (Hydrogen) till 35 (Bromine) and then decreasingly sharply at 53 (Iodine)?

**Comment by gabor1** [comment]
The E0s are calculated energies of isolated atoms using the same DFT code as the rest of the data. You can verify what they should be using Orca and calculating the isolated atom energies. Let us know if you find a discrepancy!

**Reply by JorgeG94** [reply, marked as answer]
the trend is correct although extremely non-intuitive and the internet does not have a clear answer. 

The issue here is that def2-SVP is an all electron basis set for the F, Cl, and Br atoms; however, at I it gets an ECP. So the core electrons are now being modeled by it, so the fact that the energy goes down makes a lot of sense. Only once you've panicked and thought about it. 

Short answer: it is fine. 
Long answer: Gotta take into account that ECPs exist!
